### PR TITLE
add person endpoint

### PR DESCRIPTION
**Pull request by erezmus**
closes libero/reviewer#950

### CODE REVIEW

**Comment by erezmus** [comment]
Note:

There is functionality to check the token that is repeated in the use case files: would be a good idea to move it to middleware

**Comment by diversemix** [comment]
Don't you need to change main.ts to add the GetPerson to an endpoint?

File: src/use-cases/getPerson.test.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { Option } from 'funfix';
+import * as jwt from '../jwt';
+import { Request, Response, NextFunction } from 'express';
+// import { Unauthorized, NotFound } from 'http-errors';
```
Comment (Cooryd) [inline src/use-cases/getPerson.test.ts:4]: Can we remove this commented out code?

File: src/use-cases/getPerson.test.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    rabbitmq_url: 'rabbitmq',
+    login_url: 'login-url',
+    login_return_url: 'login-return',
+    authentication_jwt_secret: 'ca-secret',
+    continuum_jwt_secret: 'continuum-secret',
+    continuum_api_url: 'somewhere',
+    knex: {} as KnexConfig,
+};
+
+describe('Get Editors', (): void => {
```
Comment (diversemix) [inline src/use-cases/getPerson.test.ts:24]: Should this be "Get Person"

File: src/use-cases/getPerson.test.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        peopleServiceMock = {
+            getPersonById: jest.fn(),
+        };
+
+        responseMock.status.mockImplementation(() => responseMock);
+
+        handler = GetPerson(config, peopleServiceMock as PeopleRepository);
+    });
+
+    it('should return with valid role', async () => {
```
Comment (diversemix) [inline src/use-cases/getPerson.test.ts:49]: Is this the correct description - as we don't appear to be getting role?

**Comment by erezmus** [comment]
ok i think i know what happened. I wanted to do some refactoring, but then abandoned it as it would be too much for PR, but accidentally removed some changes that should have gone it.

**Pull request merged**